requirements/contributing.txt

```diff
@@ -20,3 +20,10 @@ astropy-sphinx-theme
 
 # Pytest extension package enabling test coverage
 pytest-cov
+
+# For pre-commits
+pre-commit=="2.20.0" # to set hooks
```

Comment:
I had to remove the ", my pip seemed to not understand it

Reply:
yup, my bad. I also used the commit to relax bug fixes versions